gateway/src/apicast/policy/token_introspection/token_introspection.lua

```diff
@@ -77,6 +85,17 @@ local function introspect_token(self, token)
 end
 
 function _M:access(context)
+  if self.auth_type == "oidc_issuer_endpoint" then
+    if not context.proxy.oauth then
+      ngx.status = context.service.auth_failed_status
+      ngx.say(context.service.error_auth_failed)
+      return ngx.exit(ngx.status)
+    end
```

Comment:
Would be nice to have one whitespace after `end` to visually split those blocks.

Reply:
Changed.